pynest/nest/lib/hl_api_types.py

```diff
@@ -398,6 +421,10 @@ def set(self, params=None, **kwargs):
             If the input params are of the wrong form.
         KeyError
             If the specified parameter does not exist for the nodes.
+
+        See Also
+        --------
+        get
```

Comment:
I can add `GetStatus` and `SetStatus` here, but they don't get linked in the generated documentation. I'm guessing it's because they are not in the same file, so it would be the same problem the other way around. Any pointers on how to do this, @sarakonradi?

Reply:
@hakonsbm maybe the `:py:func:` works? Like here
https://github.com/nest/nest-simulator/blob/a1623206604023bbba22d486e48997d09bd93d76/pynest/nest/lib/hl_api_types.py#L527

Reply:
Almost, I had to add an explicit link as well. But apparently Sphinx didn't like mixing implicit and explicit links, so I also had to add `:py:func:` to other entries in the list. It should work now.